docs/semgrep-appsec-platform/gitlab-mr-comments.md

```diff
@@ -62,7 +62,7 @@ Creating a PAT grants the API scope to Semgrep, which lets it post comments.
 1. In GitLab, go to [<i class="fas fa-external-link fa-xs"></i> Profile > Access Tokens](https://gitlab.com/-/profile/personal_access_tokens), and then add a token with `api` scope.
 1. Copy the token created in the previous step.
 
-Once you have a GitLab PAT, you can provide it to Semgrep through [Semgrep AppSec Platform's**<i class="fa-solid fa-gear"></i> Settings > Source Code Managers** tab](/deployment/connect-scm#gitlab-self-managed-plans). This tab is a central location that makes it easy for your teams to access and rotate the token if necessary. This is the recommended method, though there are two other options for providing your PAT to Semgrep:
+Once you have a GitLab PAT, you can provide it to Semgrep through [Semgrep AppSec Platform's **<i class="fa-solid fa-gear"></i> Settings > Source Code Managers** tab](/deployment/connect-scm#gitlab-cloud). This tab is a central location that makes it easy for your teams to access and rotate the token if necessary. This is the recommended method, though there are two other options for providing your PAT to Semgrep:
 
 - In the [Network Broker configuration file](/semgrep-ci/network-broker#configure-semgrep-network-broker): If the Gitlab user requires a PAT, you can set a dummy token and assign the PAT in the Network Broker configuration file.
```

Comment:
I know this line didn't change, but as I was reading through to get the experience of the whole doc, I realized I don't know what "If the GitLab user requires a PAT" means here. What are we trying to express with that?

Reply:
I changed this line a bit, because the intent was to say that people could provide their token here and manage it locally

Reply:
yep the new one looks good, ty!